0.data_analysis/nbconverted/plates_1_2_sig_test_nf1_sc_all_cellprofiler.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Obnibus and post hoc testing
+
+# ## Imports
+
+# In[1]:
+
+
+import sys
+from sklearn.ensemble import IsolationForest
+import scikit_posthocs as sp
+import pathlib
+import numpy as np
+import pandas as pd
+
+rel_root = pathlib.Path("..")
+sys.path.append(f'{rel_root}/utils')
```

Comment:
You don't need to use pathlib here, you can keep it simple as a string. It would look like this:

```python
sys.path.append('../utils')
import analysis_utils as au
import preprocess_utils as ppu
```

Reply:
Added